templates/template.sql

```diff
@@ -1,5 +1,6 @@
 PRAGMA foreign_keys=OFF;
 BEGIN TRANSACTION;
+DROP TABLE IF EXISTS col;
```

Comment:
See https://github.com/ewnd9/anki-apkg-export/pull/8#discussion_r91520706

I've asked 

```js
export default function(deckName) {
  return new ApkgExporter(deckName, { sql, template });
}
// exporter.js
constructor() {
  ...
  this.db = new sql.Database();
  ...
}
```

Now you have 

```js
const db = new sql.Database();

export default function (deckName) {
  return new Exporter(deckName, {
    db,
    template
  });
}
```

Database is simply never reinitialized